File: trunk/nebula2/code/contrib/npythonserver/bin/stdlight.py
#-------------------------------------------------------------------------------
#   Defines the standard lighting for npyviewer.exe
#-------------------------------------------------------------------------------

stdLightTransform = new("ntransformnode", "/usr/scene/stdlight")
stdLight = new("nlightnode", "/usr/scene/stdlight/l")
stdLight.setposition(1000, 500, 1000)
stdLight.setvector("LightDiffuse",   1, 1, 1, 1)
stdLight.setvector("LightSpecular",  1, 1, 1, 1)
stdLight.setvector("LightAmbient",   1, 1, 1, 1)
stdLight.setvector("LightDiffuse1",  0.3, 0.3, 0.3, 1.0)
stdLight.setvector("LightSpecular1", 0.3, 0.3, 0.3, 1.0)

File: nebula2/code/contrib/nclodterrain/bin/stdlight.lua
-------------------------------------------------------------------------------
--   Defines the standard lighting for nviewer.exe
-------------------------------------------------------------------------------

pushcwd('.')

sel('/usr/scene')

new('ntransformnode','stdlight')
    sel('stdlight')
    new('nspotlightnode', 'l0')
	l = lookup('l0')
	sel('l0')
	l:setposition(130,30,20)
	l:seteuler(-30,45,0)
	l:setorthogonal(1000,1000,0.25,175)
	l:setvector('LightDiffuse',1,1,1,1)
	l:setvector('LightSpecular',1,1,1,1)
	l:setvector('LightAmbient',0,0,0,0)
	l:settexture('LightModMap','textures:system/white.dds')
    sel('..') 
sel('..') 

popcwd()




